src/main/java/com/gachigang/ontherun/model/dto/DepartmentDto.java

```diff
@@ -0,0 +1,23 @@
+package com.gachigang.ontherun.model.dto;
+
+import com.gachigang.ontherun.persistence.entity.Business;
+import com.gachigang.ontherun.persistence.entity.Department;
+import com.gachigang.ontherun.persistence.entity.User;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+
+/**
+ *  DTO class to represent {@link Department}.
+ */
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class DepartmentDto {
```

Comment:
We have to use DTOs insted of entities (Only in dtos), because of recurcive fetching during whole transaction in Spring.